lib/rtrace/index.d.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * The RTrace interface represents a collection of properties that help develoeprs describe the
+ * reference counts and state of the Web Assembly module. It is used by the AssemblyScript loader
```

Comment:
Isn't really used by the loader, but more of an extra package to include as an import, independently of whether the loader is used or not :)

Reply:
The line was simply removed. All set for review.